src/PHPixie/Debug.php

```diff
@@ -4,8 +4,16 @@
 
 class Debug
 {
+    /**
+     * @var \PHPixie\Debug\Builder
+     */ 
     protected $builder;
     
+    /**
+     * Instance
```

Comment:
I'm not sure this description which is basically the variable name adds any value